libraries/core_libs/storage/src/storage.cpp

```diff
@@ -81,7 +88,7 @@ DbStorage::DbStorage(fs::path const& path, uint32_t db_snapshot_each_n_pbft_bloc
 }
 
 void DbStorage::rebuildColumns(const rocksdb::Options& options) {
-  rocksdb::DB* db;
+  std::unique_ptr<rocksdb::DB> db;
```

Comment:
Same here. Why change to "std::unique_ptr<rocksdb::DB> db"?

Reply:
same as before, so in case code change, unique_ptr will delete object